api/serializers/site.js

```diff
@@ -1,47 +1,74 @@
 const yaml = require('js-yaml');
+const { omitBy, pick } = require('../utils');
 const { Site, User } = require('../models');
-const userSerializer = require('../serializers/user');
+const userSerializer = require('./user');
 const { siteViewLink, hideBasicAuthPassword } = require('../utils/site');
 
-const toJSON = (site) => {
-  const object = Object.assign({}, site.get({
-    plain: true,
-  }));
+const allowedAttributes = [
+  'id',
+  'demoBranch',
+  'demoDomain',
+  'defaultBranch',
+  'domain',
+  'engine',
+  'owner',
+  'publishedAt',
+  'repository',
+  's3ServiceName',
+  'awsBucketName',
+  'Users',
+];
+
+const dateFields = [
+  'createdAt',
+  'updatedAt',
+];
+
+const yamlFields = [
+  'defaultConfig',
+  'demoConfig',
+  'previewConfig',
+];
+
+const viewLinks = {
+  demoViewLink: 'demo',
+  previewLink: 'preview',
+  viewLink: 'site',
+};
 
-  delete object.site_users__user_sites;
-  delete object.config; // may contain sensitive info ie: basicAuth
+// Eventually replace `serialize`
+function serializeNew(site, isAdmin = false) {
```

Comment:
For future context, maybe we're more semantic with `isAdmin` ie (`isPlatformAdmin`, `isFederalistAdmin`) so we're not to be confused with the repository admin.

Reply:
Though `isAdmin` might be obsolete soon if we let folks set container config. 🤷

Reply:
updating to `isSystemAdmin`